cylc/flow/scripts/remove.py

```diff
@@ -18,7 +18,26 @@
 
 """cylc remove [OPTIONS] ARGS
 
-Remove one or more task instances from a running workflow.
+Remove task instances from a running workflow and the workflow's history.
```

Comment:
Just wondering how to clarify this for users (what are "task instances"?). How about:
```suggestion
Remove tasks from the active window (n=0) and from the workflow's history.
```

Reply:
Now outdated.